main.js

```diff
@@ -15,13 +19,132 @@ navLink.forEach((n) => n.addEventListener('click', () => {
   navMenu.classList.remove('active');
 }));
 
-formSubmit.addEventListener('submit', (e) => {
-  const emailFormat = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+function getFormInput() {
+  const data = {
+    name: userName.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('FormInput', JSON.stringify(data));
+}
+
+form.addEventListener('input', () => {
+  getFormInput();
+});
 
+form.addEventListener('submit', (e) => {
+  const emailFormat = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
   if (email.value.match(emailFormat)) {
     errorMessage.innerHTML = '';
   } else {
     e.preventDefault();
-    errorMessage.innerHTML = 'Email is invalid, must be in this format example@xyz.com';
+    errorMessage.innerHTML = 'Email is incorrect, field should not be empty. <br/> should be in this format example@xyz.com';
+  }
+});
+
+const projectInfo = [
```

Comment:
The project container has a white background color for all the view ports. So please follow the design.
Design page:
![project1](https://user-images.githubusercontent.com/35504149/136657918-e434d819-879c-4f5e-a0bd-ae476f178d6a.png)
Yours:
![yours](https://user-images.githubusercontent.com/35504149/136657932-4ec00571-d121-4b3e-8e97-d9eea9f67f9c.png)